Modules/Devices/src/Devices.Application/Identities/Commands/CreateIdentity/CreateIdentityCommand.cs

```diff
@@ -6,6 +6,7 @@ namespace Backbone.Modules.Devices.Application.Identities.Commands.CreateIdentit
 public class CreateIdentityCommand : IRequest<CreateIdentityResponse>
 {
     public string ClientId { get; set; }
+    public string TierId { get; set; }
```

Comment:
Did you really think it's a good idea that an Identity can decide which Tier it wants to be in?